helmfile/values/falco/falco-common.yaml.gotmpl

```diff
@@ -54,263 +57,36 @@ falcoctl:
       enabled: false
 
 customRules:
-  {{- if .Values.falco.customRules }}
-    {{ toYaml .Values.falco.customRules | nindent 2}}
-  {{- end }}
-  ssh-trafic.yaml: |-
-    - rule: Inbound SSH Connection
-      desc: Detect Inbound SSH Connection
-      condition: >
-        ((evt.type in (accept,listen) and evt.dir=<) or
-          (evt.type in (recvfrom,recvmsg))) and ssh_port
-      output: >
-        Inbound SSH connection (user=%user.name client_ip=%fd.cip client_port=%fd.cport server_ip=%fd.sip)
-      priority: WARNING
-      tags: [network]
-    - rule: Outbound SSH Connection
-      desc: Detect Outbound SSH Connection
-      condition: >
-        ((evt.type = connect and evt.dir=<) or
-          (evt.type in (sendto,sendmsg))) and ssh_port
-        and not (k8s.ns.name = argocd-system and k8s.pod.name startswith argocd-repo-server)
-      output: >
-        Outbound SSH connection (user=%user.name server_ip=%fd.sip server_port=%fd.sport client_ip=%fd.cip)
-      priority: WARNING
-      tags: [network]
   overwrites.yaml: |-
     - macro: k8s_containers
```

Comment:
This macro looks like it is not appending to the existing macro and you do not seem to include all of the images that are in the upstream macro. Is that intentional? I think I'd prefer if we appended to the upstream macro unless there are things there that you do not want included.

Reply:
I think the only reason is that it wasn't appended [previously](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/falco/falco-common.yaml.gotmpl#L80-L83). But i think it makes sense to append it, so we'll do that instead.